contrib/php8.3/template.py

```diff
@@ -328,6 +347,20 @@ def _ext(self):
     _extension(_extn, _iif)
 
 
+@subpackage(f"php{_majver}-pear")
+def _pear(self):
+    self.pkgdesc = f"PHP{_majver} Extension and Application Repository"
+    self.depends = [f"{pkgname}={pkgver}-r{pkgrel}", f"{pkgname}-xml"]
+    self.install_if = [f"{pkgname}={pkgver}-r{pkgrel}"]
+
+    return [
+        f"etc/php{_majver}/pear.conf",
+        f"usr/bin/pear{_majver}",
+        f"usr/bin/peardev{_majver}",
+        f"usr/bin/pecl{_majver}",
```

Comment:
this leaves behind a usr/share/php8.3/pear/ in the main package. think you have to move that too

Reply:
also not sure if /usr/share/php8.3/pear/test/ is required or if you want to add an uninstall for that above

Reply:
also, when the same version is not the default, it seems to not work:  

```
$ pear8.3
exec: /usr/bin/php: not found
```

so i think this will break when there is more than one php version. looking at the script it's because it hardcodes a /usr/bin/php

Reply:
same goes for peardev/pecl of course